console/package.json

```
@@ -7,6 +7,7 @@
 	"scripts": {
 		"prepare-config": "cpy --no-overwrite --rename=config.js config.sample.js .",
 		"serve": "npm run prepare-config && enact serve",
+		"serve:fakeTime": "set REACT_APP_FAKE_TIME=true&& npm run serve",
```

```suggestion
		"serve-fakeTime": "set REACT_APP_FAKE_TIME=true&& npm run serve",
```

I am making this suggestion just to keep the same name pattern for scripts